docs/UserGuide.md

```diff
@@ -3,11 +3,39 @@ layout: page
 title: User Guide
 ---
 
-FriendDex is a **relationship management tool for CLI enthusiasts** looking to enhance their social life while not compromising on getting things done quickly. Managing your relationship goals should not be any more tedious than coding.
+Welcome to **FriendDex**, a **relationship management tool** looking to enhance your social life while not compromising on getting things done quickly. 
+Managing your relationships should not be any more tedious than doing your laundry.
+
+Although optimized for Command Line Interface enthusiasts, it should be easy enough for anyone to use it.
```

Comment:
Copy nit: "Although optimized for Command Line Interface enthusiasts, the intuitive interface allows anyone to get started with it immediately."